server/src/index.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-console */
+import app from './app';
+import database from './database';
+
+const port:Number = app.get('port');
+
+app.listen(port, () => {
+  console.log(`server is running on http://localhost:${port}`);
+});
+
+database.sync({ force: true })
+  .then(() => console.log('Database created successfully'))
+  .catch(console.log);
```

Comment:
you are trying to sync the database on every connection
you don't need to do that
you can make a file called `sync.ts` in the database folder
and you can then make a script in your `package.json` file to execute this file